tools/tree-viz.html

```diff
@@ -8,9 +8,9 @@
     <script type="importmap">
       {
         "imports": {
-          "@grafana/lezer-logql": "https://unpkg.com/@grafana/lezer-logql@0.0.11/index.es.js",
-          "@lezer/lr": "https://unpkg.com/@lezer/lr@0.15.8/dist/index.js",
-          "@lezer/common": "https://unpkg.com/@lezer/common@0.15.11/dist/index.js"
+          "@grafana/lezer-logql": "https://unpkg.com/@grafana/lezer-logql@0.0.13/index.es.js",
+          "@lezer/lr": "https://unpkg.com/@lezer/lr@1.0.0/dist/index.js",
```

Comment:
Actually, this `@lezer/lr": "https://unpkg.com/@lezer/lr@1.0.0/dist/index.js` specific change breaks the tree viz tool. It gives `module import failed. this requires a browser with import map support.`. The issue is present from `@lezer/lr@16.0.0` and `15.8` is the last working version for this. 
<img width="1474" alt="image" src="https://user-images.githubusercontent.com/30407135/174560993-f2310b2d-8a54-4ade-8fa2-79177a94e6b0.png">